src/views/Home.jsx

```diff
@@ -20,11 +22,21 @@ export class Home extends Component {
   render() {
     return (
       <div>
-        <h1>Athletic Tracker App</h1>
+        <div className="page-title">Athletic Tracker App</div>
 
-        <h2><Link to="/status">Status Log</Link></h2>
-        <h2><Link to="/profile">Profile</Link></h2>
-        <button onClick={this.handleOnClick}>Logout</button>
+        <Link to="/status">
+        <Paper 
+          children = {"Status Log"}
+          className = "link-paper"
+        />
```

Comment:
You should be able to set your tabs to be spaces (soft-tabs). That would be preferable